packages/vx-tooltip/src/tooltips/TooltipWithBounds.tsx

```diff
@@ -64,7 +57,12 @@ function TooltipWithBounds({
 
   return (
     <Tooltip
-      style={{ top: 0, transform: `translate(${left}px, ${top}px)`, ...style }}
+      style={{
+        top: 0,
+        transform: `translate(${left}px, ${top}px)`,
+        ...style,
```

Comment:
I think for API consistency between the two components this should probably be `...(!unstyled && style)`